lib/libzfs/libzfs_pool.c

```diff
@@ -4099,7 +4099,9 @@ zbookmark_mem_compare(const void *a, const void *b)
 
 /*
  * Retrieve the persistent error log, uniquify the members, and return to the
- * caller.
+ * caller. Return nvlist_t is a list of corrupted objects. Each entity in
+ * nvlist_t contains four elements dataset number, object number, an array of
```

Comment:
Add colon to four elements ("four elements:")

Reply:
corrected